test/net-to-gross/calcInsurance.test.js

```diff
@@ -0,0 +1,73 @@
+const assert = require('assert');
+const calcTotalInsurance = require('../../src/net-to-gross/calcTotalInsurance');
+const test = require('mocha').it;
+
+
+describe('Calculate Insurances from Net and Taxes', async function () {
+
+    test('Verify maximum SI, HI, and UI are applied ' +
```

Comment:
Test case name should start with noun. Because:
The function name is 'Test' which is a verb, then Test something....
For e.g: Test('total equals 0') is better than Test('Verify total equals 0') (two verbs are next to each other)

Reply:
Check all the tests